### DISCUSSION TITLE
Inject (or otherwise manage) a collection-wide custom value via the Control Panel

### DISCUSSION THREAD

**Discussion by bruceontheloose** [statamic/cms#5234] [Q&A] [answered]
Hello, all,

One of our collections has a heading that'll be the same on every entry in the collection. We're looking for a way to:

* set the heading's text at the collection level rather than at the entry level
* have it be editable via the Control Panel so our client can change it

Is there a way to achieve this?

I do know about using [`inject`](https://statamic.dev/collections#inject) in a collection's YAML config file, but as far as I know, you can only set it by touching the code; it isn't editable in the Control Panel. It'd be great if the Configure Collection screen (`/cp/collections/my_collection/edit`) had a field for injecting a custom value(s). (We have an additional, similar use case for this feature.)

Thanks in advance for any feedback. Based on what you say, I can submit a feature request.

p.s. If necessary, we could make a separate collection with one entry whose sole purpose is for managing the heading text on the other collection. And then display that in the template used by the other collection's entries. Not elegant, but it should work.

**Comment by jasonvarga** [comment, marked as answer]
No need to ask for permission to open a feature request, feel free to just do it.

https://github.com/statamic/ideas

But yes, a CP interface for managing `inject` would be handy.

**Reply by bruceontheloose** [reply]
Thanks a lot for the quick reply, @jasonvarga. I'll open a feature request in just a little bit. (I hadn't done that initially in case my question here resulted in a solution that I was unaware of related to `inject`.)

Thanks again!
